content/en/docs/tasks/observability/distributed-tracing/opentelemetry/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: OpenTelemetry
+description: Learn how to configure the proxies to send OpenTelemetry traces to a Collector.
+weight: 10
+keywords: [telemetry,tracing,opentelemetry,span,port-forwarding]
+aliases:
+ - /docs/tasks/telemetry/distributed-tracing/opentelemetry/
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+After completing this task, you will understand how to have your application participate in tracing with [OpenTelemetry](https://www.opentelemetry.io/), regardless of the language, framework, or platform you use to build your application.
+
+This task uses the [Bookinfo](/docs/examples/bookinfo/) sample as the example application and the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) as the receiver of traces.
+
+To learn how Istio handles tracing, visit this task's [overview](../overview/).
+
+Istio can be configured to export OpenTelemetry protocol (OTLP) traces via gRPC and HTTP. Below you will find instructions
+for both scenarios.
+
+## Deploy OpenTelemetry Collector
+
+1.  Create a namespace for the Collector:
+
+{{< text bash >}}
+$ kubectl create namespace otel-collector
+{{< /text >}}
```

Comment:
```suggestion
    {{< text bash >}}
    $ kubectl create namespace otel-collector
    {{< /text >}}
```

Reply:
Fixed in [6e77f44](https://github.com/istio/istio.io/pull/14566/commits/6e77f4410d9b032ea3ba8ef05ee2b4ad3401ef37)